core/src/main/resources/org/apache/spark/ui/static/historypage.js

```diff
@@ -78,6 +78,12 @@ jQuery.extend( jQuery.fn.dataTableExt.oSort, {
     }
 } );
 
+jQuery.extend( jQuery.fn.dataTableExt.ofnSearch, {
+    "appid-numeric": function ( a ) {
+        return a.replace(/[\r\n]/g, " ").replace(/<.*?>/g, "");
```

Comment:
@ajbozarth are you saying this is an altnerative / better way to accomplish this? sorry I don't have much context to understand the comment. Or is the current approach OK?